pkg/rid/store/cockroach/subscriptions.go

```diff
@@ -52,11 +71,17 @@ func (c *subscriptionRepo) process(ctx context.Context, query string, args ...in
 			&cids,
 			&s.StartTime,
 			&s.EndTime,
+			&writer,
 			&s.Version,
 		)
 		if err != nil {
 			return nil, stacktrace.Propagate(err, "Error scanning Subscription row")
 		}
+		if writer.Valid {
```

Comment:
Ah, no need to check `writer.Valid` in this case. Instead, `s.Writer = writer.String` does the right thing as `writer.String` will be the empty string if `!writer.Valid`.

Reply:
Done. Fixed it in both cookroach/subscriptions.go and cookroach/identification_service_area.go